Remora.Discord.Commands/Attributes/DiscordDefaultMemberPermissionsAttribute.cs

```diff
@@ -0,0 +1,54 @@
+//
+//  DiscordDefaultMemberPermissionsAttribute.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using System;
+using System.Collections.Generic;
+using Remora.Discord.API.Abstractions.Objects;
+
+namespace Remora.Discord.Commands.Attributes;
+
+/// <summary>
+/// Marks a group with a set of permissions, that is the permissions required
```

Comment:
Rework this documentation comment. It doesn't read very well, and Discord still doesn't allow granular control over subcommands.